ggshield/cmd/secret/scan/changes.py

```diff
@@ -0,0 +1,59 @@
+from pathlib import Path
+from typing import Any
+
+import click
+
+from ggshield.cmd.secret.scan.secret_scan_common_options import (
+    add_secret_scan_common_options,
+    create_output_handler,
+)
+from ggshield.cmd.utils.common_decorators import exception_wrapper
+from ggshield.cmd.utils.context_obj import ContextObj
+from ggshield.core.scan import ScanContext, ScanMode
+from ggshield.core.text_utils import display_info, pluralize
+from ggshield.utils.git_shell import (
+    check_git_dir,
+    get_default_branch,
+    get_list_commit_SHA,
+)
+from ggshield.verticals.secret.repo import scan_commit_range
+
+
+@click.command()
+@add_secret_scan_common_options()
+@click.pass_context
+@exception_wrapper
+def changes_cmd(ctx: click.Context, **kwargs: Any) -> int:
+    """
+    Scan the set of changes between the current HEAD and the default branch, including staged changes.
```

Comment:
I would rewrite this as "between the default branch and the current HEAD, including staged changes". I think it reads better to have "current HEAD" and "staged changes" together.